flow/Makefile

```diff
@@ -198,9 +198,15 @@ ifeq (, $(strip $(NPROC)))
 endif
 export NUM_CORES := $(NPROC)
 
-LSORACLE_PLUGIN ?= oracle
+export LSORACLE_CMD ?= $(shell command -v lsoracle)
+ifeq ($(LSORACLE_CMD),)
+  LSORACLE_CMD = $(abspath $(FLOW_HOME)/../tools/build/LSOracle/core/lsoracle)
+endif
+
+LSORACLE_PLUGIN ?= $(abspath $(FLOW_HOME)/../tools/build/LSOracle/yosys-plugin/oracle.so)
+export LSORACLE_KAHYPAR_CONFIG ?= $(abspath $(FLOW_HOME)/../tools/build/LSOracle/core/test.ini)
```

Comment:
Maybe only set these if USE_LSORACLE is set.

Reply:
Right now the vars are only used if that flag is set, but that may not always be the case. My preference is to populate the variables in the main scope rather than relying on them being dynamically populated. This was previously discussed in the initial PR.